### PR TITLE
test: replace dealer mock with stablesats in integration tests

### PR DESCRIPTION
**Pull request by vindard**
## Description

This PR removes dealer mocks and fixes inconsistencies to allow for using stablesats in integration tests.

It also adds our Okex mock to the `make start` `make start-admin` and `make start-main` commands for local dev.

### CODE REVIEW

**Comment by vindard** [comment]
Context for these changes (https://github.com/GaloyMoney/galoy/compare/448af20...521f277), this bug took me close to 3 hours to try to dig into and isolate because the "throw" had no stack trace, it didn't stop the test execution, and the error only appeared when 2 specific test cases were run together inside the entire test suite.

File: test/integration/02-user-wallet/02-b-stablesats.spec.ts [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+    // Mid-rate values
+    expect(centsFromSatsRes.convertFromBtc.amount).toEqual(200_000n)
+    expect(centsFromSatsRes.convertFromBtcToFloor.amount).toEqual(200_000n)
+    expect(centsFromSatsRes.convertFromBtcToCeil.amount).toEqual(200_000n)
+
+    // Values with spreads
+    expect(centsFromSatsRes.getCentsFromSatsForImmediateBuy.amount).toEqual(199_800n)
+    expect(centsFromSatsRes.getCentsFromSatsForImmediateSell.amount).toEqual(200_200n)
+    expect(centsFromSatsRes.getCentsFromSatsForFutureBuy.amount).toEqual(199_760n)
+    expect(centsFromSatsRes.getCentsFromSatsForFutureSell.amount).toEqual(200_200n)
```
Comment (vindard) [inline test/integration/02-user-wallet/02-b-stablesats.spec.ts:136]: Should both `Sell` values be the same here?

Comment (vindard) [reply test/integration/02-user-wallet/02-b-stablesats.spec.ts:136]: Should we merge these changes and then circle back to the stablesats potential issue, or should we wait for that upstream to be fixed?

Comment (bodymindarts) [reply test/integration/02-user-wallet/02-b-stablesats.spec.ts:136]: I guess we can merge - @enigbe can perhaps try to reproduce this in stablesats.

**Review by bodymindarts — APPROVED** [review]

**Pull request merged**